spec/factories/tasks.rb

```diff
@@ -1,5 +1,10 @@
 FactoryBot.define do
   factory :task do
-    
+
+    sequence(:title, "title_1")
+    content { "content" }
```

Comment:
式展開しない文字列はシングルクォートで囲った方が良いですね
```suggestion
    sequence(:title, 'title_1')
    content { 'content' }
```